uint32 robot_id
uint32 pose_id
---
kimera_distributed/VLCFrameMsg frame